### ISSUE TITLE
Can't install with Angular 4.0.2

### ISSUE THREAD

**Issue by manuelbachl** [angular/flex-layout#258]
Hi there,

seems i've got some issues installing flex-layout with Angular 4.0.2

When I do an `npm install`with `@angular/flex-layout@2.0.0-beta.7` in my package.json, I get the following errors:

```
npm WARN @angular/flex-layout@2.0.0-beta.7 requires a peer of @angular/common@^2.3.5 but none was installed.
npm WARN @angular/flex-layout@2.0.0-beta.7 requires a peer of @angular/core@^2.3.5 but none was installed.
npm WARN @angular/flex-layout@2.0.0-beta.7 requires a peer of zone.js@^0.7.2 but none was installed.
```

Well, this seems just being warnings, and maybe it got installed, but this can't be true. I have a fresh Angular project with current versions of @angular/common (4.0.2), @angular/core (4.0.2) and zone.js (0.8.5)

An `ng serve` starts the project just fine, but I really want to get rid of those warnings. Is there any way?

When I perform an `npm ls` there are those warnings again (Unmet Peer Dependency):

```
npm ERR! peer dep missing: @angular/common@^2.3.5, required by @angular/flex-layout@2.0.0-beta.7
npm ERR! peer dep missing: @angular/core@^2.3.5, required by @angular/flex-layout@2.0.0-beta.7
npm ERR! peer dep missing: zone.js@^0.7.2, required by @angular/flex-layout@2.0.0-beta.7
```

**Comment by ThomasBurleson** [comment]
Refs #255

**Comment by ethanmick** [comment]
Will there be a version bump shortly after? We are waiting on this package to be updated to Angular 4 before we update everything.

Thanks!

**Comment by ThomasBurleson** [comment]
@ethanmick  - the merge to `master` should happen today or beginning of next week. Beta.8 should be released next Thursday.

**Comment by ThomasBurleson** [comment]
Fixed SHA fed87fa53850a873d2ed2b398eb3a5260a6b3e98. 
> And now available in @angular/flex-layout Beta.8 on npm.
